lib/app/widgets/banners.dart

```diff
@@ -21,8 +25,21 @@ class TokenExpiredBanner extends StatelessWidget {
     return BannerScaffold(
       backgroundColor: Colors.orange.withOpacity(0.5),
       body: Padding(
-        padding: EdgeInsets.all(8),
-        child: Text(context.s.app_error_tokenExpired),
+        padding: _bannerPadding,
+        child: Row(
+          children: <Widget>[
+            Expanded(child: Text(context.s.app_error_tokenExpired)),
+            SizedBox(width: 16),
+            IconButton(
+              onPressed: () => signOut(context),
+              icon: SvgPicture.asset(
+                'assets/icon_signOut.svg',
+                color: context.theme.isDark ? Colors.white : Colors.black,
```

Comment:
black_hole_flutter to the rescue!
```suggestion
                color: context.theme.contrastColor,
```